Client/Frontend/Browser/TabManager.swift

```diff
@@ -472,13 +472,15 @@ class TabManager: NSObject {
             return
         }
 
+        LeanplumIntegration.sharedInstance.track(eventName: .focusPromoImpression, withParameters: ["button name": buttonText as AnyObject])
         let toast = LeanplumPromoButtonToast(labelText: labelText, buttonText: buttonText, colorText: colorText, completion: { buttonPressed in
             guard let url = URL(string: deepLink) else {
                 return
             }
 
             if UIApplication.shared.canOpenURL(url) && buttonPressed {
                 UIApplication.shared.openURL(url)
+                LeanplumIntegration.sharedInstance.track(eventName: .userTappedFocusPromoButton)
```

Comment:
Do this before the openURL as that may take you away from application so LP may not track it at all ;) .